rs/ethereum/cketh/minter/src/dashboard/tests.rs

```diff
@@ -1001,8 +1001,200 @@ fn should_display_reimbursed_requests() {
         );
 }
 
+#[test]
+fn should_display_minted_events_pagination() {
+    let dashboard = {
+        let mut state = initial_state();
+        add_minted_events(&mut state, 300);
+        let paging_parameters = DashboardPagingParameters {
+            minted_events_start: 100, // Second page.
+            ..DashboardPagingParameters::default()
+        };
+        DashboardTemplate::from_state(&state, paging_parameters)
+    };
+
+    // Events are displayed in order of decreasing index. Page 2 should have events 200 to 101.
+    DashboardAssert::assert_that(dashboard)
+        .has_minted_events(
+            1,
+            &vec![
+                "0xf1ac37d920fa57d9caeebc7136fea591191250309ffca95ae0e8a7739de89cc2",
+                "200",
+                "0xdd2851Cdd40aE6536831558DD46db62fAc7A844d",
+                "ckETH",
+                "10_000_000_000_000_000",
+                "k2t6j-2nvnp-4zjm3-25dtz-6xhaa-c7boj-5gayf-oj3xs-i43lp-teztq-6ae",
+                "1",
+            ],
```

Comment:
nit: maybe extract it to a method like `minted_event_with_log_index(index: &str) -> Vec<String>` so that it's a bit clearer that what matters here is the index